site/.stylelintrc.json

```diff
@@ -9,13 +14,11 @@
     "public/css/pdf/toolbar_fullscreen.css",
     "public/css/pdf/toolbar_embedded.css",
     "public/css/pdf/pdf_embedded.css",
-    "public/css/google/source_sans_pro.css",
-    "public/css/google/pt_sans.css",
+    "public/css/google/*",
```

Comment:
We can safely ignore this whole folder. These files are auto-generated when they're pulled from google's servers.